cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/runtime/ProgramClassLoader.java

```diff
@@ -58,7 +59,7 @@
    * </pre>
    */
   public ProgramClassLoader(CConfiguration cConf, File dir, ClassLoader parent) {
-    super(cConf, dir, parent, "lib");
+    super(dir, cConf.get(Constants.AppFabric.PROGRAM_EXTRA_CLASSPATH), parent, "lib");
```

Comment:
It was hard coded inside the DirectoryClassloader earlier https://github.com/caskdata/cdap/pull/9398/files#diff-7f80e424f69c4232f806d3bcc7bbc1d1L69
I just made it configurable so that we can pass any classpath.